addons/escoria-core/game/escoria.gd

```diff
@@ -1,6 +1,7 @@
 # The escoria main script
 tool
 extends Node
+class_name Escoria
```

Comment:
Wait, does that work with the additional add_autoload_singleton in plugin? I thought you can only do one of that.

Reply:
To be honest, I can't remember at this point why I added the class name...but it appears to work? Maybe we should discuss a bit more as I don't want to do something that might introduce unintended issues.